src/lib/seam/components/SupportedDeviceTable/ShowAllDevicesButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+
+interface ShowAllDevicesButtonProps {
+  onClick: () => void
+  isShowing: boolean
+  viewingAllDevices: boolean
+  totalDeviceCount: number
+}
+
+export function ShowAllDevicesButton({
+  onClick,
+  isShowing,
+  viewingAllDevices,
```

Comment:
```suggestion
  showAllDevices,
```

Reply:
Doesn't `showAllDevices` sound more like a verb (method)? In this case it's a boolean state.

Reply:
Yea, ok how about this? These react bool props are always a bit tricky to name right. 

```
interface ShowAllDevicesButtonProps {
  onClick: () => void
  visible: boolean
  willShowAllDevices: boolean
  totalDeviceCount: number
}
```

Or maybed 'show' is the problem word. Should it be `expand / expanded` instead? 

```
interface ShowAllDevicesButtonProps {
  onClick: () => void
  visible: boolean
  expanded: boolean
  totalDeviceCount: number
}
```

Reply:
The idea is we have one idea but show/showing is odd, but expand/expanded is more natural.

Reply:
Cool, updated in https://github.com/seamapi/react/pull/251/commits/a98a477d5d2f149feb41b3d757f1f42e46e50bb5